apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/queries.ex

```diff
@@ -776,8 +775,9 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Queries do
 
     consistency = Consistency.domain_model(:read)
 
-    with n when is_number(n) or is_nil(n) <- Repo.fetch_one(query, consistency: consistency) do
-      {:ok, n}
+    with {:ok, n} when is_number(n) or is_nil(n) <-
+           Repo.fetch_one(query, consistency: consistency) do
+      n
     end
```

Comment:
```suggestion
    Repo.fetch_one(query, consistency: consistency)
```

let's not delete the `:ok | :error` tuple returned from this fetch